packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -7,7 +7,7 @@
     font-family: inherit;
     margin-bottom: var(--terra-form-checkbox-container-margin-bottom, 0);
     margin-left: var(--terra-form-checkbox-container-margin-left, 0);
-    margin-right: var(--terra-form-checkbox-container-margin-right, 2rem);
+    margin-right: var(--terra-form-checkbox-container-margin-right, 0rem);
```

Comment:
This should be 0

Reply:
Good catch, that's what I get for changing things too quickly ha.